en/lessons/advanced/escripts.md

```diff
@@ -11,11 +11,24 @@ To build executables in Elixir we will be using escript. Escript produces an exe
 
 ## Getting Started
 
-To create an executable with escript there are only a few things we need to do: implement a `main/1` function and update our Mixfile.
+For a given Mix project, to create an executable with escript there are only a few things we need to do: implement a `main/1` function and update our Mixfile.
+
+We'll begin by generating a project with a supervision tree:
+
+```shell
+$ mix new example_app
+$ cd example_app
+```
+
+We'll start by creating a module to serve as the entry point to our executable.  This is where we'll implement `main/1`. This module can be a separate file within the `lib` directory, or simply added to the default module like shown below.
```

Comment:
I'd like to keep what we had here already: "We'll start by creating a module to serve as the entry point to our executable.  This is where we'll implement `main/1`:"

By the time a reader has gotten this far in the lessons they should be familiar enough with things to know how to put this together.  We'll end up repeating too much if we review things in every lesson.